modules/structs/issue.go

```diff
@@ -143,14 +147,46 @@ type IssueFormField struct {
 // IssueTemplate represents an issue template for a repository
 // swagger:model
 type IssueTemplate struct {
-	Name     string            `json:"name" yaml:"name"`
-	Title    string            `json:"title" yaml:"title"`
-	About    string            `json:"about" yaml:"about"` // Using "description" in a template file is compatible
-	Labels   []string          `json:"labels" yaml:"labels"`
-	Ref      string            `json:"ref" yaml:"ref"`
-	Content  string            `json:"content" yaml:"-"`
-	Fields   []*IssueFormField `json:"body" yaml:"body"`
-	FileName string            `json:"file_name" yaml:"-"`
+	Name     string              `json:"name" yaml:"name"`
+	Title    string              `json:"title" yaml:"title"`
+	About    string              `json:"about" yaml:"about"` // Using "description" in a template file is compatible
+	Labels   IssueTemplateLabels `json:"labels" yaml:"labels"`
+	Ref      string              `json:"ref" yaml:"ref"`
+	Content  string              `json:"content" yaml:"-"`
+	Fields   []*IssueFormField   `json:"body" yaml:"body"`
+	FileName string              `json:"file_name" yaml:"-"`
+}
+
+type IssueTemplateLabels []string
+
+func (labels *IssueTemplateLabels) UnmarshalYAML(value *yaml.Node) error {
+	if value.IsZero() {
+		return nil
+	}
+	switch value.Kind {
+	case yaml.ScalarNode:
+		str := ""
+		err := value.Decode(&str)
+		if err != nil {
+			return err
+		}
+		for _, v := range strings.Split(str, ",") {
+			if v = strings.TrimSpace(v); v == "" {
+				continue
+			}
+			*labels = append(*labels, v)
+		}
+		return nil
+	case yaml.SequenceNode:
+		var strs []string
+		if err := value.Decode(&strs); err != nil {
+			return err
+		}
+		*labels = strs
+		return nil
+	}
+	return fmt.Errorf("line %d: cannot unmarshal %s%s into IssueTemplateLabels",
```

Comment:
`%s%s`? by purpose?

Reply:
Try to be consistent with the yaml error format:

https://github.com/go-yaml/yaml/blob/f6f7691b1fdeb513f56608cd2c32c51f8194bf51/decode.go#L362

Reply:
BUT:

![image](https://user-images.githubusercontent.com/2114189/202187291-d3833a88-9ee5-454d-9d91-28c5e98f8996.png)

Reply:
I would suggest to use `("%s %q", value.ShortTag(), value.Value)`

Reply:
I see, maybe `line %d: cannot unmarshal %s into IssueTemplateLabels` is good enough:

```text
line 3: cannot unmarshal !!map into IssueTemplateLabels
```

You can find it in the tests I just added.

Reply:
`("%s %q", value.ShortTag(), value.Value)` does work well, I'm not really sure why:

```yaml
labels:
  a: aa
  b: bb
```

```
line 3: cannot unmarshal !!map "" into IssueTemplateLabels
```